GUI/ThermoLicenceAgreementWindow.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Windows;
+
```

Comment:
is this a one-time agreement? if so, does the answer get stored?

Reply:
ya, it's a one-time agreement. it gets stored in settings.toml